test/scenarios.py

```diff
@@ -4693,6 +4693,82 @@
             ),
         },
     ),
+    Scenario(
+        "scenario26-2",
+        "Provide database with insufficient permissions, expect file share to work",
+        {
+            "ren": ActionList(
+                [
+                    action.DropPrivileges(),
+                    action.Start(
+                        "172.20.0.5",
+                        dbpath="/tmp/no-permissions/no-access-db.sqlite",
```

Comment:
Maybe `/root/no-access-db.sqlite`?

Reply:
Let me explore this tomorrow :+1:

Reply:
Done, Also changed one more test case where the `/tmp/no-permissions` was used